getData.js

```diff
@@ -0,0 +1,46 @@
+const apiUrl = "http://localhost:3000/api";
+
+const getData = (app, command, id = "") => {
+  return fetch(`${apiUrl}/${app}/${command}/${id}`, {
+    method: "GET",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  })
+    .then((response) => response.json())
+    .then((data) => {
+      return data;
+    });
+};
+
+const getBotToken = () => {
+  return getData("botSetting", "getBotToken");
+};
+
+const getHelloMessage = () => {
+  return getData("languageProfile", "getHelloMessage", "1");
+};
+
+const getLanguages = () => { 
+    return getData("languageProfile", "all")
+}
+const getAllData = () => {
+  const botToken = getBotToken().then((botToken) => {
+    return { botToken }
+  })
+  const helloMessage = getHelloMessage().then((helloMessage) => {
```

Comment:
Formatting is a bit odd here. Please add a linter to the project and follow its recommendations